userd/launcher.go

```diff
@@ -54,6 +54,11 @@ func (s *Launcher) Name() string {
 	return "io.snapcraft.Launcher"
 }
 
+// BasePath returns the base path of the object
+func (s *Launcher) BasePath() dbus.ObjectPath {
+	return "/io/snapcraft/Launcher"
+}
```

Comment:
This seems unrelated to this PR?

Reply:
This is prereq work (part of 585449b4f06b5bffea5c14fed3bd8951f7520e4f) to generalize the dbusInterface to support multiple interfaces at the same time. I can propose this as a separate PR if desired.

Reply:
No need to do that. Thanks for the explanation. :)